drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -34,9 +34,12 @@
 #include "core/os/os.h"
 #include "core/project_settings.h"
 #include "rasterizer_canvas_gles3.h"
+#include "rasterizer_storage_gles3.h"
 #include "servers/camera/camera_feed.h"
 #include "servers/visual/visual_server_raster.h"
 
+#include <iostream>
```

Comment:
iostream should not be included.

Looks like you just used it while developing, so it should be removed.